src/api/requests.ts

```diff
@@ -1,60 +1,66 @@
+import { IKartLapsPerRun, ILapDataset } from "../models/go-kart-types";
+
 // Get all sessions
-export default function getAllRuns(callback, errorCallback, finallyCallback) {
+export default function getAllRuns(
+  callback: (allRunsJSON: string[]) => void,
+  errorCallback: (error: Error) => void,
+  finallyCallback: (() => void) | undefined = undefined
+) {
   fetch(`https://go-kart-api.onrender.com/runs`)
```

Comment:
I am assuming you are yet to include rxjs, recommend building your pipeline here.

Reply:
Good comment. Thankfully he hasn't as each PR should be distinct in what it does.